src/System.Resources.ResourceManager/ref/System.Resources.ResourceManager.csproj

```diff
@@ -5,6 +5,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
     <NuGetTargetMoniker>.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.0;uap10.1</PackageTargetFramework>
```

Comment:
did you try to make this library compile for .Net Standard 1.7 and pump the version number of the contract and the the library. what exactly the validation error you are seeing?

---
In reply to: [82248278](https://github.com/dotnet/corefx/pull/12420#discussion_r82248278) [](ancestors = 82248278)